lib/sailsDbMigrate.js

```diff
@@ -82,7 +82,7 @@ function parseSailsConfig(sailsConfig) {
     throw new Error('connection missing from ./config/migrations.js');
   }
 
-  connection = sailsConfig.connections[connectionName];
```

Comment:
I think this could be something like
`connection = sailsConfig.connections ? sailsConfig.connections[connectionName] : sailsConfig.datastores ? sailsConfig.datastores[connectionName] : null;`
that way it could keep compatibility to older versions of sails